weedcoco/tests/importers/mask_data/agcontext-ginger.yaml

```diff
@@ -0,0 +1,268 @@
+### crop_type ###
+# description: 'Crop type.
+
+#   One of several strings describing the crop grown in the image.'
```

Comment:
these comments aren't needed in a test file. You can do `sed -E 's/^$|^#/d'` on the file to get only the content lines